gax-java/gax/src/main/java/com/google/api/gax/rpc/ApiClientHeaderProvider.java

```diff
@@ -41,8 +43,11 @@
 public class ApiClientHeaderProvider implements HeaderProvider, Serializable {
```

Comment:
Could we have two tests added to Showcase that show the token being added in both grpc + httpjson clients?

Reply:
added, but i notice that native-showcase is failing, would that be because the sdk jar is not included in the image yet?

Reply:
No - the native tests install the repo's modules first: https://github.com/googleapis/sdk-platform-java/blob/main/.github/workflows/ci.yaml#L272

Then starts the Showcase server, and finally runs the mvn test command: https://github.com/googleapis/sdk-platform-java/blob/main/.github/workflows/ci.yaml#L287

Reply:
fixed